docs/starter-kit-apis/passports/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Passport",
+  "position": 4
```

Comment:
should this be 5?

| SK | category.json |
| ---- | ----------- |
| DRS | position: 2 |
| WES | position: 3 | 
| Data Connect | position: 4 | 
| Passport | position: 5 |

Reply:
Also could you please edit the position value for "Data connect"? It is incorrectly set to 2